common-lib/src/main/scala/com/gu/mediaservice/lib/cleanup/ExtractGuardianCreditFromByline.scala

```diff
@@ -5,12 +5,29 @@ import com.gu.mediaservice.model.ImageMetadata
 object ExtractGuardianCreditFromByline extends MetadataCleaner {
 
   val BylineForTheGuardian = """(?i)(.+) for the (Guardian|Observer)[.]?""".r
+  val BylineForTheTrunc = """(?i)(.+) for the (.+)[.]?""".r
+
+  val GuardianPrefix = Prefix("Guardian")
+  val ObserverPrefix = Prefix("Observer")
 
   override def clean(metadata: ImageMetadata): ImageMetadata = metadata.byline match {
     case Some(BylineForTheGuardian(byline, org)) => {
       val orgName = org.toLowerCase.capitalize
       metadata.copy(byline = Some(byline), credit = Some(s"The $orgName"))
     }
+    // Catch truncated bylines (EXIF allows 32 chars only)
```

Comment:
Technically, that's IPTC's legacy `By-line` field. In EXIF it's called `Artist` and does not have said restrictions. Minor point, obv.